kubetest/aksengine.go

```diff
@@ -493,15 +493,6 @@ func (c *aksEngineDeployer) populateAPIModelTemplate() error {
 		return fmt.Errorf("No template file specified %v", err)
 	}
 
-	// set default distro so we do not use prebuilt os image
-	if v.Properties.MasterProfile.Distro == "" {
```

Comment:
setting distro from aks-engine.json would override the defaults here, right?

Reply:
That's correct.

Reply:
yes that's the path we ended up taking: https://github.com/kubernetes-sigs/windows-testing/pull/178